RecoParticleFlow/PFProducer/src/PFEGammaFilters.cc

```diff
@@ -323,6 +334,16 @@ bool PFEGammaFilters::isElectronSafeForJetMET(const reco::GsfElectron& electron,
     isSafeForJetMET = false;
   }
 
+  //TEMPORARY hack.
+  //Do not allow new EtaExtendedEle to be SafeForJetMET, until ID, regression of EEEs are in place.
+  if (!allowEEEinPF_) {
+    int nHitGsf = electron.gsfTrack()->numberOfValidHits();
+    double absEleEta = fabs(electron.eta());
+    if ((absEleEta > 2.5) && (nHitGsf < 5)) {
+      isSafeForJetMET = false;
+    }
+  }
```

Comment:
this looks like a duplication with respect to the new lines 170-176 in the same file. please write a small function that you can call in either place, to avoid code duplication.